### DISCUSSION TITLE
how to change ux_mode in vue3 ?

### DISCUSSION THREAD

**Discussion by Rachael0108** [wavezync/vue3-google-signin#39] [Q&A]
i want to change mode to redirect mode 
i use this in mobile phone, popup mode is not suitable

**Comment by kasvith** [comment]
What are you using? composables or the button?

**Reply by kasvith** [reply]
there is a `ux_mode` property for changing this for composables
for button it is `ux-mode`

**Comment by pweimann** [comment]
You can add additional properties as follows.

`
const { isReady, login } = useCodeClient({
  onSuccess: handleOnSuccess,
  onError: handleOnError,
  // other options,
  'ux_mode': 'redirect',
  'redirect_uri': 'http://example.com/code/callback'
});
`
